core/perf_test/PerfTest_ViewResize_8.cpp

```diff
@@ -47,10 +47,18 @@
 namespace Test {
 
 TEST(default_exec, ViewResize_Rank8) {
+// FIXME_SYCL Avoid running out of resources on the CUDA GPU used in the CI
+#ifdef KOKKOS_ENABLE_SYCL
+  printf("Resize View Performance for LayoutLeft:\n");
+  run_resizeview_tests8<Kokkos::LayoutLeft>(9, 1);
+  printf("Resize View Performance for LayoutRight:\n");
+  run_resizeview_tests8<Kokkos::LayoutRight>(9, 1);
+#else
```

Comment:
I'm not entirely sure why allocating a lot of memory is more problematic now, but we had similar issues already in other places when testing `SYCL+CUDA`. This might be mitigated by #4097.